packages/safe-tools-client/app/components/deposit-modal/index.gts

```diff
@@ -53,35 +53,43 @@ export default class DepositModal extends Component<Signature> {
       <BoxelActionContainer as |Section ActionChin|>
         <Section @title="Deposit Instructions" class="deposit-modal__section">
           <p>
-            It is the user’s responsibility to ensure that sufficient funds are
-            present in their wallet at the time of each transaction
+            It is the your responsibility to ensure that sufficient funds are
```

Comment:
`It is your responsibility`  there's a leftover `the`

Reply:
Thanks for catching that!